data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -240,22 +242,21 @@ def _create_tile_data(
     def _create_downloadable_data(
         self, score_county_state_merged_df: pd.DataFrame
     ) -> pd.DataFrame:
-        df = score_county_state_merged_df[constants.DOWNLOADABLE_SCORE_COLUMNS]
-
-        float_columns = df.select_dtypes(include=["float64"])
-
-        # score_tiles[constants.TILES_SCORE_FLOAT_COLUMNS] = score_tiles[
-        #     constants.TILES_SCORE_FLOAT_COLUMNS
-        # ].apply(
-        #     func=lambda series: floor_series(
-        #         series=series,
-        #         number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
-        #     ),
-        #     axis=0,
-        # )
-
-        # [x for x in df.columns if field_names.PERCENTILE_FIELD_SUFFIX in x]
-
+        df = score_county_state_merged_df[
+            constants.DOWNLOADABLE_SCORE_COLUMNS
+        ].copy(deep=True)
+
+        df_of_float_columns = df.select_dtypes(include=["float64"])
+
+        for column in df_of_float_columns.columns:
+            for item in constants.PERCENT_PREFIXES_SUFFIXES:
+                if item in column:
+                    df[column] = df[column] * 100
```

Comment:
@lucasmbrown-usds this is where we're trying to add code to cast as an integer if the value is not NaN, othewise leave as NaN

Reply:
@esfoobar-usds This is from @lucasmbrown-usds in Slack  `FYI we can also just multiply the percentiles by 100 when we create them in data/data-pipeline/data_pipeline/etl/score/etl_score.py:494`

Reply:
Should just be: 

```
from data_pipeline.etl.score.etl_utils import floor_series

...

df[column] = (floor_series(
                series=df[column] * 100,
                number_of_decimals=0,
            )).astype('Int64')
```

Reply:
Yeah, we referenced that [nullable](https://pandas.pydata.org/pandas-docs/version/0.24/user_guide/integer_na.html) integer type as you showed Lucas. I think we were still have side effects propagate in some of the columns. I can retry that again and see if that does indeed rectify some of the problems we noted.

Reply:
Got an error:

```
Exception has occurred: TypeError
data type 'Int64' not understood
  File "/opt/justice40-tool/data/data-pipeline/data_pipeline/etl/score/etl_score_post.py", line 253, in _create_downloadable_data
    df[column] = (
  File "/opt/justice40-tool/data/data-pipeline/data_pipeline/etl/score/etl_score_post.py", line 282, in transform
    self.output_downloadable_df = self._create_downloadable_data(
  File "/opt/justice40-tool/data/data-pipeline/data_pipeline/etl/runner.py", line 102, in score_post
    score_post.transform()
  File "/opt/justice40-tool/data/data-pipeline/data_pipeline/application.py", line 206, in generate_score_post
    score_post(data_source)
  File "/opt/justice40-tool/data/data-pipeline/data_pipeline/application.py", line 283, in <module>
    cli()
```

with this code:

```
                if item in column:
                    df[column] = (
                        floor_series(
                            series=df[column] * 100,
                            number_of_decimals=0,
                        )
                    ).astype("Int64")
```